web/static/sass/screen.scss

```diff
@@ -295,8 +302,16 @@ code {
     margin-top: 80px;
 
     @media #{$large-screen} {
-      position: absolute;
       bottom: 0;
+      position: absolute;
+    }
+
+    @media #{$not-tall-screen},
+           #{$not-large-screen},
+           #{$large-screen} and #{$not-tall-screen} {
```

Comment:
Why is this applied for large screens? Should't it just be for "not tall" screens?